snuba/datasets/storages/metrics.py

```diff
@@ -47,6 +49,35 @@
     Column("offset", UInt(64)),
 ]
 
+polymorphic_bucket = WritableTableStorage(
```

Comment:
Tied to @lyn's comment. Now each metric entity has a writable storage. Once this is done and we get rid of the other storages, how will that look like? Are we going to have this writable storage as an Entity-less storage (not associated to any entity)? which would be cool.
And are we going to remove the writable storage from the metrics entities? Not sure if that is possible, never tried.

Reply:
I was assuming the first option

Reply:
If I'm not mistaken what Filippo suggested are not separate options but 2 changes that are compatible. I think we should do both.
- separate writable storage (basically this PR)
- remove the other 3 writable storages from the metrics entities so they each only have a readable storage

Reply:
Is there a reason they can't share a writable storage? that could let use re-use the sentry snuba integration tests without making changes in that repo